DQM/HcalTasks/interface/HcalGPUComparisonTask.h

```diff
@@ -0,0 +1,58 @@
+#ifndef HcalGPUComparisonTask_h
```

Comment:
By the way, usually a plugin should not have the class declared in a header file under `interface`, because it cannot be used by other packages.
Either move the header file under `plugins`, or just merge it into `HcalGPUComparisonTask.cc` itself.

Reply:
merged into HcalGPUComparisonTask.cc